src/btree/row_modify.c

```diff
@@ -311,13 +311,13 @@ __wt_row_insert_alloc(WT_SESSION_IMPL *session, const WT_ITEM *key, u_int skipde
  */
 WT_UPDATE *
 __wt_update_obsolete_check(
-  WT_SESSION_IMPL *session, WT_PAGE *page, WT_UPDATE *upd, bool update_accounting)
+  WT_SESSION_IMPL *session, WT_PAGE *page, WT_UPDATE *upd, bool update_accounting, u_int *upd_count)
```

Comment:
I'd go with `uint32_t` instead of `u_int`, just out of sheer raving paranoia.

Reply:
Removed this argument from the function `__wt_update_obsolete_check`